src/internal/carriers/tracking/tracking-info.ts

```diff
@@ -18,8 +18,11 @@ export class TrackingInfo extends ShipmentIdentifierBase {
   public readonly packages: readonly PackageTrackingInfo[];
   public readonly events: readonly TrackingEvent[];
 
-  public get package(): PackageTrackingInfo {
-    return this.packages[0];
+  public get package(): PackageTrackingInfo | undefined {
+    if(this.packages && this.packages.length > 0) {
+      return this.packages[0];
+    }
+
```

Comment:
The `if` statement is unnecessary, since `this.packages` will always be an array.  All that's needed is `return this.packages[0];`